### PR TITLE
Escape é in info.js

### PR DESCRIPTION
**Pull request by Bryan-0**

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1427,7 +1427,7 @@ var commands = exports.commands = {
 			pokemon = {types: [type1.id]};
 			target = type1.id;
 		} else {
-			return this.sendReplyBox("" + Tools.escapeHTML(target) + " isn't a recognized type or pokemon.");
+			return this.sendReplyBox("" + Tools.escapeHTML(target) + " isn't a recognized type or pok&eacute;mon.");
```
Comment (Rhythms88) [inline chat-plugins/info.js]: The P should also be capitalized

Comment (Zarel) [inline chat-plugins/info.js]: Really, convention is just to keep "pokemon" with no accent when lowercase.

**Comment by TheImmortal** [comment]
>2 commits

**Comment by Bryan-0** [comment]
I know...

I'm not in home, I will be back in 1 hour.

**Comment by Bryan-0** [comment]
Updated.

**Pull request merged**